### PR TITLE
Document how to delete rack volumes

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

Addresses convox/rack#1492

### CODE REVIEW

**Comment by soulshake** [comment]
[how to clean up persistent volumes](https://trello.com/c/cMQ77zia/1470-how-to-clean-up-persistent-volumes)

**Comment by ddollar** [comment]
Let's make the example directory more descriptive, e.g.:

`/volumes/appname/processname`

**Comment by soulshake** [comment]
P.S. Addresses https://github.com/convox/rack/issues/1492

File: _deployment/volumes.md [hunk 1]
```diff
@@ -67,3 +68,15 @@ services:
 ```
 
 This configuration will work with both `convox start` and `convox deploy`. Files written to `/var/www/html` will be persisted on the Docker host.
+
+## Cleanup
+
+Occasionally volumes become corrupted. If this happens, note that you can delete volumes on a Rack via `convox instances ssh`, e.g.:
```
Comment (mattmanning) [inline _deployment/volumes.md]: "Occasionally volumes become corrupted" sounds kind of scary and like the volumes feature is inherently broken, which I don't think is the case. Can we drop the first sentence here and just start with "You can delete volumes on a Rack . . . "

Comment (soulshake) [inline _deployment/volumes.md]: I was going for searchability in cases like [the issue that inspired this PR](https://github.com/convox/rack/issues/1492), so a user can find this page when trying to figure out what to do about a corrupt volume. WDYT?

**Pull request merged**